src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -634,6 +634,16 @@ public function text(string $default = null, bool $normalizeWhitespace = true)
         return $text;
     }
 
+    /**
+     * Returns only the inner text that is the direct descendent of the current node, excluding any child nodes.
+     *
+     * @return string Inner text
```

Comment:
```suggestion
```

Reply:
I would tend to agree this isn't particularly necessary, but it is consistent with the docblock style of all other methods that return non-void in this file. Do we really want to buck that trend now, and if so, should all other `@return` annotations be removed also?

Reply:
The other docblocks are probably present because at the time the component was created we were not able to use real return type declarations. There's no need to add more for cases where the native return type is sufficient.